core/src/main/targetconfigs/vz/model.yaml

```diff
@@ -25,6 +36,7 @@ spec:
             replicas: {{{replicas}}}
```

Comment:
what is replicas ?

Reply:
Added this comment to both model.yaml files:
```
# The number of cluster member Managed Server instances to start for this WebLogic cluster
```